primedev/shared/keyvalues.cpp

```diff
@@ -1,5 +1,7 @@
 #include "keyvalues.h"
-#include <winnt.h>
```

Comment:
I assumed `winnt.h` was actually unused and hence removed?

Reply:
opposite actually
`winnt.h` cannot work without including some other stuff first.
In this case switching to `windows.h` worked just fine.